baby-gru/src/components/list/MoorhenCarbohydrateList.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useRef, useState } from "react";
+import { Row, Col } from "react-bootstrap";
+import { moorhen } from "../../types/moorhen";
+import { webGL } from "../../types/mgWebGL";
+import { useSelector } from "react-redux";
+import { getLigandSVG } from "../../utils/MoorhenUtils";
+import { MoorhenCarbohydrateCard } from "../card/MoorhenCarbohydrateCard";
+import {PrivateerResultsEntry} from "../../types/privateer";
+
+export const MoorhenCarbohydrateList = (props: {
+    setBusy?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
You probably want to call `props.setBusy(true)` and `props.setBusy(false)` at the start and at the end of `validate`. That way the spinner will show up when users open the Glycans accordion inside the molecule card.